atlasdb-config/src/main/java/com/palantir/atlasdb/http/FailoverFeignTarget.java

```diff
@@ -143,11 +147,16 @@ private void checkAndHandleFailure(RetryableException ex) {
     }
 
 
-    private void pauseForBackOff(RetryableException ex) {
-        double pow = Math.pow(
+    private void pauseForBackoff(RetryableException ex) {
+        double pauseTime = Math.pow(
                 GOLDEN_RATIO,
                 numSwitches.get() * failuresBeforeSwitching + failuresSinceLastSwitch.get());
-        long timeout = Math.min(maxBackoffMillis, Math.round(pow));
+        pauseForBackoff(ex, Math.round(pauseTime));
```

Comment:
This is pretty awesome, and we should implement it, but can also be part of a future PR if we want to get this in ASAP